dist/rpm/openQA.spec

```diff
@@ -232,7 +232,8 @@ next to the webui.
 Summary:        Convenience package for a single-instance setup
 Group:          Development/Tools/Other
 Requires:       %{name}-local-db
-Requires:       %{name}-worker
+Requires:       %{name} = %{version}
```

Comment:
so you think local-db can stay with the version-less requirement but here we should pull in the version specific version. Makes sense. I just suggest you move the explanation from the git commit message into a comment here explaining that. The functional change looks fine.